app/src/main/java/com/duckduckgo/app/browser/omnibar/QueryUrlConverter.kt

```diff
@@ -38,10 +39,10 @@ class QueryUrlConverter @Inject constructor(private val requestRewriter: Request
 
         val uriBuilder = Uri.Builder()
             .scheme(https)
-            .appendQueryParameter("q", searchQuery)
+            .appendQueryParameter(AppUrl.ParamKey.QUERY, searchQuery)
```

Comment:
we had the parameter, but it wasn't used

Reply:
Nice!